extensions/ql-vscode/src/view/model-alerts/ModelAlerts.tsx

```diff
@@ -81,6 +96,18 @@ export function ModelAlerts({
     };
   }, []);
 
+  const modelAlerts = useMemo(() => {
+    if (!repoResults) {
+      return [];
+    }
+
+    const alerts = repoResults
+      .filter((r) => r.interpretedResults !== undefined)
+      .flatMap((a) => a.interpretedResults!);
```

Comment:
Can we avoid the `!` here by doing something like this?

```suggestion
    const alerts = repoResults.flatMap((a) => a.interpretedResults ?? []);
```